### ISSUE TITLE
Connection timed out error when calling getUsage

### ISSUE THREAD

**Issue by KaracsonyMarton** [DeepLcom/deepl-php#10]
$usage = $translator->getUsage();

DeepL\ConnectionException: Operation timed out after 10001 milliseconds with 0 bytes received in /var/www/vendor/deeplcom/deepl-php/src/HttpClient.php:200

**Comment by daniel-jones-deepl** [comment]
HI @KaracsonyMarton, thanks for reporting the issue.

Does this problem occur continually? It could be just a temporary condition.

**Comment by KaracsonyMarton** [comment]
> 

Yes, before each translation, we make a call to the translate API endpoint only if we are sure we have enough credits.
